src/Function/Relation/Binary/Equality.agda

```diff
@@ -0,0 +1,37 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Definition of extensional equivalence.
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+open import Level
+open import Relation.Binary.Bundles
+open import Relation.Binary.Definitions
+
+module Function.Relation.Binary.Equality
+  {b ℓ : Level}
+  (B-setoid : Setoid b ℓ)
```

Comment:
Changed as requested in next push.

Reply:
Shoot! I think I actually need to keep `A` as a general purpose variable.
I'm getting the following w/ the new code:

```agda
66 foldr-cong : ∀ {f g : V → S → S} {d e : S} →
67              (∀ {y z} → y ≈ z → ∀ x → f x y ≈ g x z) → d ≈ e →
68             foldr f d ≗ foldr g e
69 foldr-cong f≈g d≈e []       = d≈e
70 foldr-cong f≈g d≈e (x ∷ xs) = f≈g (foldr-cong f≈g d≈e xs) x

```

(All code on line 68 highlighted in medium brown.)

```
/home/dbanas/prj/agda-stdlib/src/Algebra/Linear/Properties/VectorSpace.agda:68,14-23
V !=< List _A_210
when checking that the inferred type of an application
  List _A_210 → _B_212
matches the expected type
  Setoid.Carrier ≈ᴹ-setoid → Setoid.Carrier setoid
```

I'm guessing that if I fix this for `List` then it will just break again when I move to `Foldable Functor`.
What do you think?

I've temporarily added a new module: `Function.Relation.Binary.Equality2`, which contains the new code (i.e. - modified as per your request).
And it can be selected, via comment toggling, at these locations:

1. `src/Algebra/Linear/Properties/VectorSpace.agda: 39`
2. `src/Algebra/Module/Morphism/Bundles.agda: 50`

I think I'll need some help w/ this one if you want to push towards two module parameterization.

Thanks!